InnoTopicWebsite/src/styles.sass

```diff
@@ -110,7 +110,7 @@ app-user-other-profile-icon
 
 .otherProfileIcon
```

Comment:
@samyakjainnnnn Can you please move these css from `styles.sass`  to it's component?
Also if it is overriding the default component css than you can apply the style by selecting it's parent class.

Reply:
updated!